forui/lib/src/widgets/select_tile_group/select_tile.dart

```diff
@@ -0,0 +1,183 @@
+class FSelectTile<T> extends StatelessWidget with FTileMixin {
+  /// The style.
+  final FTileStyle? style;
+
+  /// The checked icon. Defaults to `FIcon(FAssets.icons.check)`.
+  final Widget? checkedIcon;
+
+  /// The unchecked icon.
+  final Widget? uncheckedIcon;
+
+  /// The title.
+  final Widget title;
+
+  /// The subtitle below the title.
+  final Widget? subtitle;
+
+  /// The details on the right hand side of the title and subtitle.
+  final Widget? details;
+
+  /// The semantic label used by accessibility frameworks.
+  final String? semanticLabel;
+
+  /// The current value.
+  final T value;
+
+  /// Called when the user initiates a change to the [FSelectTile]'s value: when they have checked or unchecked this tile.
+  final ValueChanged<bool>? onChange;
+
+  /// Whether this radio tile is enabled. Defaults to true.
+  final bool? enabled;
+
+  /// Whether this radio tile should focus itself if nothing else is already focused. Defaults to false.
+  final bool autofocus;
+
+  /// Defines the [FocusNode] for this radio.
+  final FocusNode? focusNode;
+
+  /// Handler called when the focus changes.
+  ///
+  /// Called with true if this widget's node gains focus, and false if it loses focus.
+  final ValueChanged<bool>? onFocusChange;
+
+  final Widget? _icon;
+
+  final bool _suffix;
+
+  /// Creates a [FSelectTile] with a prefix check icon.
+  FSelectTile({
+    required this.title,
+    required this.value,
+    this.style,
+    this.subtitle,
+    this.details,
+    this.semanticLabel,
+    this.onChange,
+    this.enabled,
+    this.autofocus = false,
+    this.focusNode,
+    this.onFocusChange,
+    Widget? checkedIcon,
+    Widget? uncheckedIcon,
+    Widget? suffixIcon,
+    super.key,
+  })  : checkedIcon = checkedIcon ?? FIcon(FAssets.icons.check),
+        // We use a transparent color instead of a sized box, to simplify calculation of the checked icon size.
+        uncheckedIcon = uncheckedIcon ?? FIcon(FAssets.icons.check, color: Colors.transparent),
+        _suffix = false,
+        _icon = suffixIcon;
+
+  /// Creates a [FSelectTile] with a suffix check icon.
+  FSelectTile.suffix({
+    required this.title,
+    required this.value,
+    this.style,
+    this.subtitle,
+    this.details,
+    this.semanticLabel,
+    this.onChange,
+    this.enabled,
+    this.autofocus = false,
+    this.focusNode,
+    this.onFocusChange,
+    Widget? prefixIcon,
+    Widget? checkedIcon,
+    Widget? uncheckedIcon,
+    super.key,
+  })  : _icon = prefixIcon,
+        checkedIcon = checkedIcon ?? FIcon(FAssets.icons.check),
+        // We use a transparent color instead of a sized box, to simplify calculation of the checked icon size.
+        uncheckedIcon = uncheckedIcon ?? FIcon(FAssets.icons.check, color: Colors.transparent),
+        _suffix = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final FSelectTileData(:controller, :selected) = FSelectTileData.of<T>(context);
```

Comment:
So a `FSelectTile` must exist inside a `FSelectGroupTile`? If so, I think the docs is quite misleading as it gave the impression that a `FSelectTile` can be used stand-alone. Would it be better to just place everything under the same `FSelectGroupTile` page?

Reply:
That's fair.